app.rb

```diff
@@ -0,0 +1,84 @@
+require_relative 'database_connection_setup'
+require 'sinatra'
+require 'sinatra/flash'
+require_relative './lib/user'
+require_relative './lib/chitter'
+
+class ChitterWebApp < Sinatra::Base
+
+  enable :sessions, :method_override
+  register Sinatra::Flash
+
+  get '/' do
+    erb(:login)
+  end
+
+  post '/feed' do
+    if User.find_user(params[:username]) == false
+      flash[:username] = 'username does not exist'
+      redirect '/'
+    elsif User.login_correct?(params[:password]) == false
+      flash[:password] = 'incorrect password'
+      redirect '/'
+    else
+      redirect '/feed'
+    end
+  end
+
+  get '/feed' do
+    @user = User.current_user
+    @peeps = Chitter.all
+    erb(:feed)
+  end
+
+  post '/new_peep' do
```

Comment:
The rubric is also suggesting grouping methods into controller groups - a group for user related routes, for peeps, etc. See below:

<img width="394" alt="Screenshot 2020-06-08 at 10 37 24" src="https://user-images.githubusercontent.com/63285668/84015622-0bf30d80-a974-11ea-9b11-0d47f6ce8b99.png">